Lib/test/test_dict.py

```diff
@@ -212,6 +213,15 @@ def __next__(self):
 
         self.assertRaises(ValueError, {}.update, [(1, 2, 3)])
 
+    def test_init(self):
```

Comment:
That's unrelated to this PR, right?

Reply:
It's mostly unrelated but I assumed that these tests are good to have anyway. Should I remove them? (when you ask a question like that, it's not always easy to understand whether it's just a question or whether you want something to be fixed).